user/cli11/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "cli11"
+pkgver = "2.5.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCLI11_SINGLE_FILE=OFF",  # Build the CLI11.hpp file from the sources. Requires Python (version 3 or 2.7).
+    "-DCLI11_PRECOMPILED=ON",  # generate a precompiled static library instead of header-only
```

Comment:
setting `CLI11_PRECOMPILED=ON` allows creation of a `cli11-devel-static` subpackage